src/config/graphical-session/graphics-drivers/nvidia.md

```diff
@@ -0,0 +1,79 @@
+# NVIDIA
+
+## nouveau (Open Source Driver)
+
+Install the Mesa DRI driver `mesa-dri` and optionally the Xorg driver
+`xf86-video-nouveau`.
+
+> Note: This is already included in the `xorg` meta-package, but is needed when
+> installing via `xorg-minimal`.
+
+This driver is developed mostly by the community, with little input from Nvidia,
+and is not as performant as the proprietary driver. It is required in order to
+run most Wayland compositors.
```

Comment:
This para should probably be moved up, as the first para under the "nouveau" heading.